src/plot_map.py

```diff
@@ -0,0 +1,32 @@
+import pandas as pd
+import geopandas as gpd
+import matplotlib.pyplot as plt 
+from src.data_ingestion import create_path_to_data
```

Comment:
When you are inside a package (src), you can use relative paths (from .data_ingestion import ...)